hack/cloud-image-uploader/azure.go

```diff
@@ -0,0 +1,420 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
```

Comment:
this is a nit, but we need to apply general Go formatting here, probably easier to set up an editor to do that for you automatically (most modern IDEs do it out of the box).

also we probably need to set up linting process to lint such files.